modules/ocf_backups/files/create-encrypted-backup

```diff
@@ -23,30 +39,40 @@ fi
 cleanup() {
     if [ -e "$SNAPSHOT_PATH" ]; then
         echo "Removing logical volume: $SNAPSHOT_PATH"
-        lvremove -f "$SNAPSHOT_PATH"
+        /sbin/lvremove -f "$SNAPSHOT_PATH"
```

Comment:
maybe we should change the cron PATH globally via puppet? seems same to always put sbin on the path (and avoids trouble when debian moves things, e.g. we had trouble when kadmin moved)